app/assets/javascripts/discourse/app/components/d-modal.hbs

```diff
@@ -55,9 +55,9 @@
                     {{/if}}
                   </div>
                 {{/if}}
-
-                {{yield to="headerBelowTitle"}}
```

Comment:
This broke the modal tabs styling.

Before the fix / After the fix

<img width="572" alt="Screenshot 2023-08-03 at 16 33 03" src="https://github.com/discourse/discourse/assets/66961/8e278110-2588-4d5d-b184-33eb4fa3c89a">

<img width="573" alt="Screenshot 2023-08-03 at 16 31 58" src="https://github.com/discourse/discourse/assets/66961/2eb9a540-2efb-4549-95d9-671d87e4ae18">